### PR TITLE
Alternative HTTP remote port and path

### PR DESCRIPTION
**Pull request by dehy**
Added ability to customize remote path and port.
I actually found the issue while load-balancing Tomcat Webapp. I had an app server at port 8080 and app at path /javaapp.

### CODE REVIEW

File: scripts/startup.py [hunk 1]
```diff
@@ -157,18 +157,25 @@ def parse_env(env=os.environ):
         m = link_pattern.match(var)
         if m:
             service_name = m.group('service_name')
+            service_port = int(m.group('service_port'))
```
Comment (jasonwyatt) [inline scripts/startup.py:160]: Any reason in particular you're casting to an `int`, versus checking if `service_port != '80'` later?

Comment (dehy) [inline scripts/startup.py:160]: No particular reason, no.

**Comment by jasonwyatt** [comment]
Looks good to me. :)  Thanks!

**Comment by dehy** [comment]
Thanks to you for your original code! :)

**Pull request merged**